templates/Calculate_Mixing_Scores.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>Is there a reason we need to calculate this separately? <code>p1_ratio = 1 / p2_ratio.</code>  So can we set a single max value? Or take the log2 and set an absolute min/max? If it's actually easier to do it the current way don't need to change, but I think it would simplify the code and have fewer parameters for the user. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>